### ISSUE TITLE
Cannot view components

### ISSUE THREAD

**Issue by DMaxter** [vuejs/devtools-next#318]
I'm using version 7.0.25 of the DevTools and Vue 3 and currently it gives me this error when trying to view the components:

```
Uncaught (in promise) TypeError: j[0] is undefined
    B http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    f http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    promise callback*setup/f/< http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    f http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    setup http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    Ei http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    Ei http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    setup http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    wg http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    bAA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    mAA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    Z http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    O http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    h http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    rA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    run http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:15
    update http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    wg http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    iU http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    promise callback*rU http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    cQ http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    effect http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    U1 http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:15
    U4 http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:15
    Bu http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:15
    set value http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:15
    k http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    F http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    promise callback*F http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    v http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    g http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:27
    AeA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:23
    wg http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    Gn http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    Gn http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    A http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:23
    ao http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:23
    qAA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:23
    eeA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:23
    D http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    p http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    h http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    rA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    run http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:15
    update http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    eA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    Z http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    O http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    h http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    S http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    k http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    h http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    S http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    k http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    h http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    S http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    k http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    h http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    S http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    D http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    p http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    h http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    rA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    run http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:15
    update http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
    eA http://localhost:5173/__devtools__/assets/index-De6CzrZK.js:19
```

This is message is shown everytime I try to open the Components tab in the DevTools, so it is quite unusable at the moment

**Comment by webfansplz** [comment]
Can you provide a mini repo? 🙏

**Comment by DMaxter** [comment]
I am working with grpc-web and it seems that the issue starts to occur when I add the generated proto files. The reproducer contains code from my repo but since I had basically just started, I think it is good enough

(This reproducer makes calls to a backend but I'm not providing it, however it is not important as the error occurs with and without the backend)

[reproducer.zip](https://github.com/vuejs/devtools-next/files/14893414/reproducer.zip)

**Comment by webfansplz** [comment]
Fixed in 7.0.26
